packages/react-docsite-components/src/utilities/activePage.ts

```diff
@@ -1,65 +1,77 @@
 import { INavPage } from '../components/Nav/index';
-import { removeAnchorLink } from './removeAnchorLink';
```

Comment:
The changes to this file are:
- using the `getNormalizedPath` helper to ensure consistent comparisons
- adding a variant of the original `hasActiveChild` function which returns the active page object itself (for use in `Site.tsx`)